source/retryable-writes/retryable-writes.rst

```diff
@@ -66,10 +66,50 @@ ClientSession
    name of this object MAY vary across drivers.
 
 Retryable Error
-   Any network exception or a write command response with an error indicating
-   that the node is no longer a primary (e.g. "not master" or "node is
-   recovering" errors). Such errors correspond to those discussed in the SDAM
-   spec's section on `Error Handling`_.
+   An error is considered retryable if it meets any of the following criteria:
```

Comment:
Something happened to the rendering of this file after these changes. Open this link in a browser:

https://github.com/kevinAlbs/specifications/blob/spec1051/source/retryable-writes/retryable-writes.rst

Compare to:

https://github.com/mongodb/specifications/blob/master/source/retryable-writes/retryable-writes.rst

Reply:
Oh good catch, that was because of the inconsistent heading for the "What do the additional error codes mean?". I also updated the table spacing for consistency. Looks like it's fixed now.